src/App.css

```diff
@@ -1,3 +1,240 @@
-.app {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@300;500;700;800&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --black: var(--black);
+  --medium-blue: #307bbe;
+  --pale-grey: #f5f6fa;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --z: #fafafa;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+  --font-M: montserrat, arial, helvetica, sans-serif;
+  --font-R: robotoslab, arial, helvetica, sans-serif;
+}
+
+body {
+  background-color: var(--pale-grey);
+  padding: 8% 2% 7% 6%;
+}
+
+.main {
+  margin: 0 2% 4% 0;
+  padding: 0 0 16%;
+  border-radius: 4px;
+  box-shadow: 10px 10px 10px 10px rgba(0, 0, 0, 0.05);
+  background-color: var(--z);
+}
+
+.navigation {
+  margin: 0 0 2%;
+  padding: 1% 6% 1% 6%;
+  background-color: var(--white-two);
+  border: solid 1px var(--white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
- [OPTIONAL] Good job implementing all the suggested changes by the previous reviewer. however, there are still some instances that were missed by him and also the core point in his review was to follow the DRY principle and to avoid repeating the same rule over and over again. so instead of giving `display: flex` on so many places, what you can do is to make a class like this:
```
.flex {
display: flex;
}
```
and then give this `flex` class to any element you want in the JSX. That way if you apply this in all the other instances in your styling, then your code would be aligned with the DRY principle.

Reply:
Thanks, I'll implement it.